### PR TITLE
Aion Web3 improvements

### PR DESCRIPTION
**Pull request by aionWilliam**
Updated links in readme files of each package,
Updated tests for aion unit conversion
Migrated etherum unit conversion to aion-unit conversion implementations and functions
Updated file headers to reflect Aion License

### CODE REVIEW

File: packages/web3-utils/src/utils.js [hunk 1]
```diff
@@ -1,26 +1,27 @@
...
+ *
+ *     You should have received a copy of the GNU General Public License
+ *     along with the aion network project source files.  
+ *     If not, see <https://www.gnu.org/licenses/>.
+ *
+ * Contributors:
+ *     Aion foundation.
+ *     Marek Kotewicz <marek@parity.io>
+ *     Fabian Vogelsteller <fabian@frozeman.de>
+ */var _ = require('underscore');
```
Comment (aion-kelvin) [inline packages/web3-utils/src/utils.js]: can you put a new line before "var" ?

File: packages/web3/src/index.js [hunk 2]
```diff
@@ -1,28 +1,26 @@
...
+ *     See the GNU General Public License for more details.
+ *
+ *     You should have received a copy of the GNU General Public License
+ *     along with the aion network project source files.  
+ *     If not, see <https://www.gnu.org/licenses/>.
+ *
+ * Contributors:
+ *     Aion foundation.
+ *     Marek Kotewicz <marek@parity.io>
+ *     Fabian Vogelsteller <fabian@frozeman.de>
```
Comment (aion-kelvin) [inline packages/web3/src/index.js]: the original file had a few other Ethereum contributors, can we add those guys too?

File: packages/web3-utils/src/bloomFilter.js [hunk 3]
```diff
@@ -1,27 +1,27 @@
...
+ *     See the GNU General Public License for more details.
+ *
+ *     You should have received a copy of the GNU General Public License
+ *     along with the aion network project source files.  
+ *     If not, see <https://www.gnu.org/licenses/>.
+ *
+ * Contributors:
+ *     Aion foundation.
+ *     Marek Kotewicz <marek@parity.io>
+ *     Fabian Vogelsteller <fabian@frozeman.de>
```
Comment (aion-kelvin) [inline packages/web3-utils/src/bloomFilter.js]: Let's add the author from the original file too

File: packages/web3-providers-http/src/index.js [hunk 4]
```diff
@@ -1,25 +1,26 @@
...
+ *     See the GNU General Public License for more details.
+ *
+ *     You should have received a copy of the GNU General Public License
+ *     along with the aion network project source files.  
+ *     If not, see <https://www.gnu.org/licenses/>.
+ *
+ * Contributors:
+ *     Aion foundation.
+ *     Marek Kotewicz <marek@parity.io>
+ *     Fabian Vogelsteller <fabian@frozeman.de>
```
Comment (aion-kelvin) [inline packages/web3-providers-http/src/index.js:24]: let's add that one missing contributor

File: packages/web3-core-requestmanager/src/jsonrpc.js [hunk 5]
```diff
@@ -1,25 +1,26 @@
...
+ *     See the GNU General Public License for more details.
+ *
+ *     You should have received a copy of the GNU General Public License
+ *     along with the aion network project source files.  
+ *     If not, see <https://www.gnu.org/licenses/>.
+ *
+ * Contributors:
+ *     Aion foundation.
+ *     Marek Kotewicz <marek@parity.io>
+ *     Fabian Vogelsteller <fabian@frozeman.de>
```
Comment (aion-kelvin) [inline packages/web3-core-requestmanager/src/jsonrpc.js:23]: there's a few contributors from the old file that aren't in this one, can we add those guys?

**Review by aion-kelvin — CHANGES_REQUESTED** [review]
thanks for working on this!  just a few small things

**Comment by aionWilliam** [comment]
Fixed the commented issues, also went through all the src files and updated contributors in file header.

**Review by aion-kelvin — APPROVED** [review]

**Comment by aion-kelvin** [comment]
looks good!

**Pull request merged**